src/main/kotlin/com/numberone/daepiro/domain/address/entity/Address.kt

```diff
@@ -1,14 +1,41 @@
 package com.numberone.daepiro.domain.address.entity
 
+import com.numberone.daepiro.domain.address.utils.AddressUtils
 import com.numberone.daepiro.domain.baseentity.PrimaryKeyEntity
+import jakarta.persistence.CascadeType
 import jakarta.persistence.Entity
+import jakarta.persistence.OneToMany
 import jakarta.persistence.Table
+import org.hibernate.annotations.Comment
 
 @Entity
 @Table(name = "`address`")
 class Address(
+    @Comment("시/도")
     val si: String,
+
+    @Comment("시/군/구")
     val gu: String? = null,
+
+    @Comment("읍/면/동")
     val dong: String? = null,
-    val depth: Int = 0
-) : PrimaryKeyEntity()
+
+    // 1: 시/도, 2: 시/군/구, 3: 읍/면/동
+    val depth: Int = 0,
+
+    @OneToMany(mappedBy = "address", cascade = [CascadeType.ALL])
+    val userAddresses: List<UserAddress> = emptyList()
+) : PrimaryKeyEntity() {
+    companion object {
+        fun of(
```

Comment:
인자를 하나 받는 경우 from 어때요? 
```suggestion
        fun from(
```